source/fundamentals/builders.txt

```diff
@@ -19,35 +19,76 @@ Builders
    :depth: 2
    :class: singlecol
 
-This section includes guides on how to use each of the available builders, and demonstrates the utility the Java driver
-builder classes provide.
+This section includes guides on how to use each of the available
+builders, and demonstrates the utility the Java driver builder classes
+provide. 
 
 Overview
 --------
 
-The Java driver provides classes to make it easier to use CRUD operations and the Aggregation API. The static utility methods
-allow you to build a query more succinctly.
+The Java driver provides classes to simplify the process for developers
+to use CRUD operations and the Aggregation API without having to refer
+to MongoDB reference documentation. The static utility methods allow you
```

Comment:
nitpick: is this true? I'm not sure we should state that builders are to help the user avoid looking at documentation

Reply:
I was trying to convey you wouldn't have to look at it as often, but I agree, it isn't something we should state.